parser.go

```diff
@@ -716,7 +716,7 @@ func (uoe *UnsupportedOptionError) Error() string {
 }
 
 // ParseRule parses an IDS rule and returns a struct describing the rule.
```

Comment:
Sorry, GolangCI closed so automatic linting is busted...
This comment should be `// parseRuleAux...` now